RELEASE.md

```diff
@@ -0,0 +1,114 @@
+  not.
+- If any existing files have been changed, API-breaking changes have been made,
+  and a major release is required.
+- If new files have been added (i.e. new protocols have been added), the change
+  is still API preserving (a client from the same major release family can
+  connect and communicate successfully without needing to know about the new
+  protocols). This indicates new functionality, and a minor release is
+  required.
+- If files have been deleted, API-breaking changes have been made (this is a
+  special case of existing files being changed).
+
+### Process if the current release is a prerelease
+
+If the codebase is currently in a prerelease state, we don't want to repeat
+increments of the same kind. E.g. If a previous prerelease has already
+incremented the `patch` number, do not increment it again. If the `minor`
+number has already been incremented, do not increment it or the `patch` version
+again. If the `major` version has already been incremented, do not change
+increment anything.
+
+However, if a larger increment is required than has already been applied,
+update the version number. For example, if the `minor` number has already been
+incremented, and a breaking change is added, increment the `major` number.
+
+## Step 2: Create a Pre-Release Tag
+
+Tag the latest commit on `main` with a pre-release tag. A pre-release tag has
+the following format:
+
+```
+<component>@v<major>.<minor>.<patch>-prerelease<n>
+```
+
+where `<component>` is the name of the component being released. Currently, the
+release process is defined for the `runtime` component, but other components
+could be added, such as the validator or the language. These releases will always
+be accompanied by a release of the runtime however.
+
+The `major`, `minor`, and `patch` numbers should match what is in the `cabal`
+files, and `n` is a monotonically increasing number that gets bumped between
+successive prerelease tags.
+
+Push this tag to github with `git push --tags`. Do this directly on `main`.
+
+## Step 3: Ensure Nightly Tests Have Run
+
+Run the nightly tests on the tagged commit if they have not already been run on
+that commit. If they fail, file a bug report and abort the current release.
+After the failures have been addressed, start again at step 1 (following the
+"current prerelease" process).
+
+## Step 4: Manual QA
+
+Schedule or perform manual QA of the changes listed in the change log since the
+last release. Also perform basic smoke tests to check for regressions. If
+manual QA finds any issues, file a bug report and abort the current release.
+After the issues have been addressed, start again at step 1 (following the
+"current prerelease" process).
+
+## Step 5: Create a Release Tag
+
+Create and push a release tag *on the same commit as the prerelease tag*.
+A release tag is the same as a prerelease tag with the `-prerelease<n>` suffix
+dropped.
+
+## Step 6: Publish Haskell Libraries to CHaP
+
+TODO
```

Comment:
This should conform to https://github.com/input-output-hk/cardano-engineering-handbook/blob/5cc0176c81404e21bf55b33a6e2bed4101b00d68/policy/haskell/packaging/versioning.md.